test/.solhintignore

```diff
@@ -0,0 +1 @@
+*_style_guide_rst.sol
```

Comment:
The Style guide file is challenging to check right now for a variety of reasons, so will just ignore for now.